DataConnectors/Senserva/SenservaPro.json

```diff
@@ -0,0 +1,159 @@
+			"query": "let timeframe = 60d;\n            SenservaPro_CL\n            | where TimeGenerated >= ago(timeframe)\n            | where ControlName_s == 'AzureSecureScorePWAgePolicyNew'"
+		},
+		{
+			"description": "All 'AzureSecureScoreRoleOverlap' controls received in the last 60 days",
+			"query": "let timeframe = 60d;\n            SenservaPro_CL\n            | where TimeGenerated >= ago(timeframe)\n            | where ControlName_s == 'AzureSecureScoreRoleOverlap'"
+		},
+		{
+			"description": "All 'AzureSecureScoreSelfServicePasswordReset' controls received in the last 60 days",
+			"query": "let timeframe = 60d;\n            SenservaPro_CL\n            | where TimeGenerated >= ago(timeframe)\n            | where ControlName_s == 'AzureSecureScoreSelfServicePasswordReset'"
+		},
+		{
+			"description": "All 'AzureSecureScoreSigninRiskPolicy' controls received in the last 60 days",
+			"query": "let timeframe = 60d;\n            SenservaPro_CL\n            | where TimeGenerated >= ago(timeframe)\n            | where ControlName_s == 'AzureSecureScoreSigninRiskPolicy'"
+		},
+		{
+			"description": "All 'AzureSecureScoreUserRiskPolicy' controls received in the last 60 days",
+			"query": "let timeframe = 60d;\n            SenservaPro_CL\n            | where TimeGenerated >= ago(timeframe)\n            | where ControlName_s == 'AzureSecureScoreUserRiskPolicy'"
+		},
+		{
+			"description": "All 'Disabled' controls received in the last 60 days",
+			"query": "let timeframe = 60d;\n            SenservaPro_CL\n            | where TimeGenerated >= ago(timeframe)\n            | where ControlName_s == 'Disabled'"
+		},
+		{
+			"description": "All 'NonAdminGuest' controls received in the last 60 days",
+			"query": "let timeframe = 60d;\n            SenservaPro_CL\n            | where TimeGenerated >= ago(timeframe)\n            | where ControlName_s == 'NonAdminGuest'"
+		},
+		{
+			"description": "All 'ServicePrincipalNotUsingClientCredentials' controls received in the last 60 days",
+			"query": "let timeframe = 60d;\n            SenservaPro_CL\n            | where TimeGenerated >= ago(timeframe)\n            | where ControlName_s == 'ServicePrincipalNotUsingClientCredentials'"
+		},
+		{
+			"description": "All 'StaleLastPasswordChange' controls received in the last 60 days",
+			"query": "let timeframe = 60d;\n            SenservaPro_CL\n            | where TimeGenerated >= ago(timeframe)\n            | where ControlName_s == 'StaleLastPasswordChange'"
+		}
+	],
+	"dataTypes": [
+		{
+			"name": "SenservaPro_CL",
+			"lastDataReceivedQuery": "SenservaPro_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		}
+	],
+	"connectivityCriterias": [
+		{
+			"type": "IsConnectedQuery",
+			"value": [
+				"SenservaPro_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+			]
+		}
+	],
+	"availability": {
+		"status": 0,
```

Comment:
Status should be 2 .Please update the status.

Thanks,
Aditi Maheshwari

Reply:
@v-admahe This change has been made